pkg/services/subscriptions/README.md

```diff
@@ -0,0 +1,33 @@
+The API response for /subscriptions look as follows (p.s. the uuids are fake, so don't try to hack the planet with these).
+If you'd like to make your own API requests, you can follow this guide to get you started:
+
+https://medium.com/@mauridb/calling-azure-rest-api-via-curl-eb10a06127
+
+`GET`:
+
+```sh
+➜ curl -s -X GET -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" "https://management.azure.com/subscriptions?api-version=2020-01-01" | jq .
+{
```

Comment:
Good idea to have this in a README. As a reminder to us for later, I also still like the idea of having a package or dir with the raw JSON responses to use for tests/quick instantiation.